### PR TITLE
[FIX] Fixed some issues at the isochrone color picker

### PR DESCRIPTION
**Pull request by Ebubeker**
There were some issues at the isochrone color picker. The isochrone colors couldn't be reset after closing the dialogue and reopening it. Also another issue is that if you left the dialog open and clicked another isochrone, the dialogue wouldn't update therefore you did not know which isochrone you were editing. So basically I added an if statement which checks if there is any dialog open. If yes then the selection will not be made.

To have a better understanding you could also check the code

### CODE REVIEW

**Comment by EPajares** [comment]
@Ebubeker thank you very much! Let's discuss this in the meeting later.

File: app/client/src/components/isochrones/IsochroneColorPicker.vue [hunk 1]
```diff
@@ -184,6 +183,11 @@ export default {
       this.calculationColors = newColors;
     },
     resetStyle(calculation) {
+      console.log(
```
Comment (EPajares) [inline app/client/src/components/isochrones/IsochroneColorPicker.vue:186]: Could you remove the console.log

File: app/client/src/components/isochrones/Isochrones.vue [hunk 2]
```diff
@@ -1900,6 +1902,7 @@ export default {
         });
     },
     removeCalculation(calculation) {
+      console.log(calculation);
```
Comment (EPajares) [inline app/client/src/components/isochrones/Isochrones.vue:1905]: same

**Comment by Ebubeker** [comment]
The reason is that I am currently working at the #1664 issue so maybe we can wait till that is also finished and we can push them together.

**Comment by Ebubeker** [comment]
The issues have been fixed. And I have also removed the console logs, hopefully there is no more left of them.

**Comment by Ebubeker** [comment]
Now everything is finished and ready for review.

**Pull request merged**